openquake/commonlib/logictree.py

```
@@ -797,27 +934,117 @@ def validate_uncertainty_value(self, node, branchset, value):
                 'expected a pair of floats separated by space'
             )
         elif branchset.uncertainty_type == 'incrementalMFDAbsolute':
-            mbr = value.split(',')
-            if len(mbr) == 3:
-                min_mag, bin_width, rates = mbr
-                try:
-                    rates = valid.positivefloats(rates)
-                except ValueError:
-                    rates = []
-                if _float_re.match(min_mag) and _float_re.match(bin_width) and\
-                        len(rates):
-                    return
+            min_mag, bin_width = (node.incrementalMFD["minMag"],
+                                  node.incrementalMFD["binWidth"])
+            rates = node.incrementalMFD.occurRates.text.strip()
+            try:
+                rates = valid.positivefloats(rates)
+            except ValueError:
+                rates = []
```

Are we hiding the error here? If an user inserts wrong rates instead of getting an error the rates mysteriously
disappear?